### ISSUE TITLE
Help!  I can't eval using weight-sparse learner.

### ISSUE THREAD

**Issue by yupanpanwangrui** [Tencent/PocketFlow#127]
After compress the model resnet on dataset cifar10 using weight-sparse learner, I want to evaluate the accuracy. I set --exec_mode=eval, but the result is follow:

INFO:tensorflow:model files: model.ckpt-97656.data-00000-of-00001, checkpoint, model.ckpt-97656.meta, model.ckpt-97656.index
INFO:tensorflow:Restoring parameters from ./models/model.ckpt-97656
INFO:tensorflow:starting 0-th roll-out
INFO:tensorflow:a-loss = 0.00e+00 | c-loss = 0.00e+00 | noise std. = 1.00e+00
INFO:tensorflow:Restoring parameters from ./models_pruned/model.ckpt

This is not normal．I have the folder models_ws, but it is compress again when i set --exec_model=eval.

Any help and hint is appreciated.

**Comment by jiaxiang-wu** [comment]
Sorry, it is a bug in our implementation. We should skip the RL-based optimization process when `exec_mode` is set to “eval”. We will fix it ASAP.

**Comment by jiaxiang-wu** [comment]
Resolved in PR #131.
